glados-web/src/routes.rs

```diff
@@ -214,22 +214,28 @@ pub async fn content_dashboard(
             StatusCode::INTERNAL_SERVER_ERROR
         })?;
 
-    let audits_of_recent_content: Vec<AuditTuple> =
-        get_audits_for_recent_content(KEY_COUNT, &state.database_connection).await?;
-
-    let recent_audits: Vec<AuditTuple> =
-        get_recent_audits(KEY_COUNT, &state.database_connection).await?;
-    let recent_audit_successes: Vec<AuditTuple> =
-        get_recent_audit_successes(KEY_COUNT, &state.database_connection).await?;
-    let recent_audit_failures: Vec<AuditTuple> =
-        get_recent_audit_failures(KEY_COUNT, &state.database_connection).await?;
+    // Run queries for content dashboard data concurrently
+    let results = tokio::join!(
+        get_audits_for_recent_content(KEY_COUNT, &state.database_connection),
+        get_recent_audits(KEY_COUNT, &state.database_connection),
+        get_recent_audit_successes(KEY_COUNT, &state.database_connection),
+        get_recent_audit_failures(KEY_COUNT, &state.database_connection),
+        get_audit_stats(Period::Hour, &state.database_connection),
+        get_audit_stats(Period::Day, &state.database_connection),
+        get_audit_stats(Period::Week, &state.database_connection),
+    );
+
+    // Get results from queries
+    let audits_of_recent_content: Vec<AuditTuple> = results.0?;
+    let recent_audits: Vec<AuditTuple> = results.1?;
+    let recent_audit_successes: Vec<AuditTuple> = results.2?;
+    let recent_audit_failures: Vec<AuditTuple> = results.3?;
+    let hour_stats = results.4?;
+    let day_stats = results.5?;
+    let week_stats = results.6?;
```

Comment:
*nit* these high index numbers feel a little awkward, and an opportunity for bugs as the code evolves. I think it is a silght improvement to destructure up at the join and then self-reference to unwrap the errors, like:
```suggestion
    let audits_of_recent_content: Vec<AuditTuple> = audits_of_recent_content?;
    let recent_audits: Vec<AuditTuple> = recent_audits?;
    let recent_audit_successes: Vec<AuditTuple> = recent_audit_successes?;
    let recent_audit_failures: Vec<AuditTuple> = recent_audit_failures?;
    let hour_stats = hour_stats?;
    let day_stats = day_stats?;
    let week_stats = week_stats?;
```

That saves us from some slightly annoying bulk line changes in the future, if we need to remove, for example, the 2nd element from the joined tasks.